defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 # defaults file for oulibraries.dspace6
+dspace6_default_java_opt_setting: '-Xmx512m -XX:MaxPermSize=256m'
```

Comment:
I might go with "dspace6_java_opts" to have a shorter name. I do like prefixing role variables with the role name (as you have done). After that, I try for a short name that matches closely with the variable that I'm replacing.

Reply:
done.